samples/net/coaps_client/README.rst

```diff
@@ -17,13 +17,13 @@ command:
 
 .. code-block:: console
 
-	make server
+	$ make server
```

Comment:
FYI, I agree with adding the prompt $ here.   A general rule of thumb is if you'd expect the user to cut and paste a complex command (or a few lines of commands), then leave off the prompt $ since that would make it hard to cut and paste.